app/components/primary_navigation_component.html.erb

```diff
@@ -1,11 +1,11 @@
-<div class="moj-primary-navigation govuk-!-display-none-print">
-  <div class="moj-primary-navigation__container">
-    <div class="moj-primary-navigation__nav">
-      <nav class="moj-primary-navigation" aria-label="Primary navigation">
-        <ul class="moj-primary-navigation__list">
+<div class="app-primary-navigation govuk-!-display-none-print">
+  <div class="app-primary-navigation__container">
+    <div class="app-primary-navigation__nav">
```

Comment:
I noticed that we aren't defining the following class: `app-primary-navigation__nav` in `_primary-navigation.scss 
 ` as opposed to `moj-primary-navigation__nav`, is this because of it forcing the left alignment of all tabs? Are there any repercussions of leaving it out of the new primary navigation?
old class definition for reference:
 ```
 .moj-primary-navigation__nav {
  text-align: left;
  @include govuk-media-query($from: desktop) {
    display: inline-block;
    vertical-align: middle;
  }

}
 ```

Reply:
Eagle eye - yep the MOJ primary navigation component has 2 parts to it. The nav items aligned left, and the search box aligned right. We don’t use the search box aligned right.

So as I didn't want to cause bloat and bring in code we aren't using, I have just renamed all the classes to `app-` etc